Client/Frontend/Browser/TabManager.swift

```diff
@@ -306,14 +295,6 @@ class TabManager: NSObject {
         return tab
     }
 
-    fileprivate func addTab(_ request: URLRequest? = nil, configuration: WKWebViewConfiguration? = nil, afterTab: Tab? = nil, flushToDisk: Bool, zombie: Bool) -> Tab {
```

Comment:
Thanks for removing this! This was done in the old Obj-C style and was never updated!